update_db.txt

```diff
@@ -1,2 +1,2 @@
-0 9,15,21 * * * ./update_campus.sh
-@weekly ./update_ctown.sh
+0 9,15,21 * * * cd /usr/src/app; /usr/local/bin/python update_campus.py >> /var/log/cron.log 2>&1
+@weekly cd /usr/src/app; /usr/local/bin/python update_ctown.py >> /var/log/cron.log 2>&1
```

Comment:
what is `2>&1` for?

Reply:
This allows error messages to also be logged cause before it was suppressed. https://stackoverflow.com/questions/818255/in-the-shell-what-does-21-mean